### ISSUE TITLE
Add cpfp_index and change_index to onchain_txs response

### ISSUE THREAD

**Issue by edouardparis** [revault/revaultd#318]
When client is retrieving the information, it is hard to understand which outputs are the recipients, the change or the cpfp

Inspiration: `listspendtxs` https://github.com/revault/revaultd/blob/master/src/daemon/jsonrpc/api/mod.rs#L1296

**Comment by darosior** [comment]
Yeah. We already do a hack for `listspendtxs`. We should make it part of revault_tx. https://github.com/revault/revaultd/pull/333#issuecomment-996624861

**Comment by darosior** [comment]
https://github.com/revault/revault_tx/pull/119
